cgmanifest.json

```diff
@@ -21629,6 +21740,17 @@
         }
       }
     },
+    {
+      "component": {
+        "type": "other",
+        "other": {
+          "name": "python-trio",
+          "version": "0.16.0",
+          "downloadUrl": "https://github.com/python-trio/trio/archive/v0.16.0.tar.gz"
+        }
+      }
+    },
+    
```

Comment:
Nit: empty line. I'd suggest running `toolkit/scripts/update_cgmanifest.py last cgmanifest.json SPECS-EXTENDED/[spec_name]/[spec_name].spec` from the repo's root. This will read the URL from the spec and put the information in correct order into `cgmanifest.json`. You can also run it with multiple spec paths as arguments.

Reply:
Good catch. Fixed.